config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl

```diff
@@ -0,0 +1,15 @@
+INSERT DATA {
```

Comment:
I had some issues with this migration and I still haven't figured it out. After the migration ran (the migration service marked it as done) the data wasn't actually added in the database which meant the form option wasn't visible. After copying this query in Virtuoso's SPARQL interface it did show work so it's not the query itself.

Reply:
@Windvis does this issue still persist? I think it's due to the file extension being defined as `.ttl` instead off `.sparql`. It is weird that the migrations service wouldn't complain, that I do admit.

Reply:
Doh, that will be it for sure, thanks. I haven't retested it to be honest, I didn't create a backup of a "clean db" and running the migrations from 0 takes a very long time.

Reply:
:arrow_up: yup, that's seems to do the trick!

Reply:
Thanks!